### PR TITLE
fix: shared post card long lines

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- By adding `flex` - we override the `display: -webkit-box` from the `line-clamp-*`.
- Removed flex to make the class work.
- Also changed the div to p to be semantically correct.

Quick link: https://preview3.app.daily.dev/squads/rustdevs

Preview:
Screenshot 2024-03-06 at 6 55 33 PM

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-208 #done

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]

**Comment by omBratteng** [comment]
image

File: packages/shared/src/components/cards/SharedPostCardFooter.tsx [hunk 1]
```diff
@@ -26,14 +26,14 @@ export const SharedPostCardFooter = ({
         isShort ? 'flex-row items-center' : 'flex-col',
       )}
     >
-      <div
+      <p
         className={classNames(
-          'flex text-theme-label-secondary typo-footnote',
```
Comment (rebelchris) [inline packages/shared/src/components/cards/SharedPostCardFooter.tsx:31]: Does this not give issues for the short posts then?

**Review by rebelchris — APPROVED** [review]
Non blocking but maybe good to. check?

File: packages/shared/src/components/cards/SharedPostCardFooter.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/cards/SharedPostCardFooter.tsx:31]: I did ran through multiple Squads, such as AI and Rush which have a wide variety of posts. They all seem to work okay.

**Pull request merged**